README.md

```diff
@@ -75,7 +75,7 @@ Prerequisites for building Flink:
 ```
 git clone https://github.com/apache/flink.git
 cd flink
-mvn clean package -DskipTests # this will take up to 10 minutes
+./mvnw clean package -DskipTests # this will take up to 10 minutes
```

Comment:
Switched to recommend using the project provided Maven wrapper. Later versions of Maven installed on a developer's computer will fail since the use of insecure repositories was depreciated.